### PR TITLE
Create new project and initiate log functions

### PR DESCRIPTION
**Pull request by Robchaves**
New project created and some functions defined

### CODE REVIEW

File: localMatchApp/localMatchApp.pro [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# depend on your compiler). Please consult the documentation of the
+# deprecated API in order to know how to port your code away from it.
+DEFINES += QT_DEPRECATED_WARNINGS
+
+# You can also make your code fail to compile if you use deprecated APIs.
+# In order to do so, uncomment the following line.
+# You can also select to disable deprecated APIs only up to a certain version of Qt.
+#DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x060000    # disables all the APIs deprecated before Qt 6.0.0
+
+CONFIG += c++11
```
Comment (wfallascr) [inline localMatchApp/localMatchApp.pro]: Change this to 14

File: localMatchApp/log.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#include "log.h"
+
+using namespace std;
+
+log::log()
```
Comment (wfallascr) [inline localMatchApp/log.cpp]: Change the name of this module to avoid confusion with debug log

**Review by wfallascr — CHANGES_REQUESTED** [review]

File: localMatchApp/logIn.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#include "logIn.h"
+
+using namespace std;
+
+logIn::logIn()
```
Comment (luvalverde) [inline localMatchApp/logIn.cpp:5]: according with the coding standard, it is necessary to include:
1.	Destructor
2.	Copy constructor
3.	Copy Assignment operator

All of them should be implemented.

File: localMatchApp/logIn.cpp [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+{
+    //Connect to database.
+}
+
+logIn::~logIn()
+{
+    //disconnect to database.
```
Comment (luvalverde) [inline localMatchApp/logIn.cpp:12]: Maintain consistence with all the comments.
All other comments start with uppercase

File: localMatchApp/logIn.h [hunk 5]
```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include <string>
+
+
+class logIn
```
Comment (luvalverde) [inline localMatchApp/logIn.h:5]: .h files must include all functions documentation (in doxygen)

File: localMatchApp/localMatchApp.pro (hunk 1 above)
Comment (Robchaves) [reply localMatchApp/localMatchApp.pro]: Changed to 14

File: localMatchApp/log.cpp (hunk 2 above)
Comment (Robchaves) [reply localMatchApp/log.cpp]: changed to logIn

File: localMatchApp/logIn.h [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+{
+public:
+  logIn();
+  ~logIn();
+
+  int signUp(const std::string user, const std::string password);
```
Comment (luvalverde) [inline localMatchApp/logIn.h:11]: Arguments whose input value is used by the function, and whose value is not changed by the function should be prefixed by "i_" (i.e. input).  
Apply this comment everywhere

File: localMatchApp/main.cpp [hunk 7]
```diff
@@ -0,0 +1,11 @@
+#include "mainwindow.h"
+#include <QApplication>
+
+int main(int argc, char *argv[])
+{
+  QApplication a(argc, argv);
+  MainWindow w;
```
Comment (luvalverde) [inline localMatchApp/main.cpp:7]: let's use more representative names

File: localMatchApp/main.cpp [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+  w.show();
```
Comment (luvalverde) [inline localMatchApp/main.cpp:8]: comments would be useful

**Review by luvalverde — CHANGES_REQUESTED** [review]
please apply these changes

File: localMatchApp/logIn.h (hunk 6 above)
Comment (luvalverde) [inline localMatchApp/logIn.h:11]: it is not necessary to use std::string, use 
```c
using namespace std;
```

File: localMatchApp/logIn.cpp [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+int logIn::signUp(const string user, const string password)
+{
+  //Send information to database.
+  return 0;
+}
+
+bool logIn::isEqualPass(const string pass, const string rePass)
```
Comment (luvalverde) [inline localMatchApp/logIn.cpp:21]: let's use more representative names for the parameters

File: localMatchApp/logIn.h [hunk 10]
```diff
@@ -0,0 +1,16 @@
+#include <iostream>
```
Comment (luvalverde) [inline localMatchApp/logIn.h:1]: why do you need this?
let's only push to master code that is being used

File: localMatchApp/logIn.cpp (hunk 3 above)
Comment (Robchaves) [reply localMatchApp/logIn.cpp:5]: Done

File: localMatchApp/logIn.cpp (hunk 4 above)
Comment (Robchaves) [reply localMatchApp/logIn.cpp:12]: Done

File: localMatchApp/logIn.h (hunk 6 above)
Comment (Robchaves) [reply localMatchApp/logIn.h:11]: Done

File: localMatchApp/logIn.cpp (hunk 9 above)
Comment (Robchaves) [reply localMatchApp/logIn.cpp:21]: Done

**Pull request closed without merging**